packages/rum-recorder/src/boot/recorder.spec.ts

```diff
@@ -135,6 +135,35 @@ describe('startRecording', () => {
       expectNoExtraRequest(done)
     })
   })
+
+  it('takes a full snapshot when the view changes', (done) => {
+    const { lifeCycle } = setupBuilder.build()
+
+    lifeCycle.notify(LifeCycleEventType.VIEW_CREATED, {} as any)
+
+    flushSegment(lifeCycle)
+
+    waitRequests(2, (requests) => {
+      expect(requests[0].data.get('has_full_snapshot')).toBe('true')
```

Comment:
FMU, the first request corresponds to the segment that start at the beginning of the recording and end with the creation of the new view.
If that is it, do we need to ensure that it has a full snapshot in this test?

Reply:
No, I'll remove it.